README.md

```diff
@@ -5,34 +5,32 @@
 
 # Knot.x Fragments
 
-> While [Knot.x HTTP Server](https://github.com/Knotx/knotx-server-http) is a _"hearth"_ of Knot.x, Fragments processing 
-is its _"brain"_.
+> While [Knot.x HTTP Server](https://github.com/Knotx/knotx-server-http) is a _"hearth"_ of Knot.x, 
+> Fragments processing is its _"brain"_.
 
-Knot.x Fragments is a Swiss Army knife for **integrating with dynamic data sources**. It 
-comes with **distributed systems stability patterns** such as a **circuit breaker** to handle different 
-kinds of network failures. Thanks to those build-in mechanisms you can focus more on delivering
-business logic and be ready to handle any unexpected integration problems.
+**Knot.x Fragments is a Swiss Army knife for **integrating with various data sources**. It provides 
+instruments that transform business use cases into implementation logic ready for evolution.**
 
-Knot.x Fragments encourages to decompose business logic into a chain of simple steps that later 
-can be wrapped with integration stability patterns without code changes. 
-Besides, when the chain becomes more complex and additional failure scenarios are known, 
-failure logic can be adjusted with fallback configuration (no changes in the business logic required).
+Knot.x Fragments is designed to build fault-tolerant, reactive, back-end integrations such as:
+- [API Gateway](https://github.com/Knotx/knotx-example-project#api-gateway--web-api)
+- Backend For Frontend (BFF) for single-page applications (SPA)
+- [Web API](https://github.com/Knotx/knotx-example-project#api-gateway--web-api) (both REST and GraphQL)
 
-Knot.x Fragments is designed to build fault-tolerant, back-end integrations such as:
-- API Gateways, Backend For Frontend (BFF) for single-page applications (SPA), Web APIs
-- documents processing (HTML, JSON, PDF etc) with a templating engine support
+Its initial purpose was a [templating solution](https://github.com/Knotx/knotx-example-project#template-processing) 
```

Comment:
I feel like as someone who wants to know what Fragments are I'm more interested in what they are now. Maybe add a sentence or two clarifying how this purpose shifted